docs/csharp/codedoc.md

```diff
@@ -12,16 +12,464 @@ ms.devlang: dotnet
+        /// </listheader>
+        /// <item>
+        /// <term>Col 1 Row 1</term>
+        /// <term>Col 2 Row 1</term>
+        /// <term>Col 3 Row 1</term>
+        /// </item>
+        /// <item>
+        /// <term>Col 1 Row 2</term>
+        /// <term>Col 2 Row 2</term>
+        /// <term>Col 3 Row 2</term>
+        /// </item>
+        /// </list>
+        /// </summary>
+        public void DoALotMoreStuff(params string[] args)
+        {
+
+        }
+    }
+
+<a name="see"></a>
+## &lt;see&gt;
+
+The &lt;see&gt; tag lets you specify a link from within text. Use the `cref` attribute to create internal hyperlinks to documentation pages for code elements.
+
+    /// <summary>
+    /// This class does something
+    /// </summary>
+    public class SomeClass
+    {
+        /// <summary>
+        /// This method does stuff
+        /// See <see cref="SomeClass.DoMoreStuff" /> to do more stuff
+        /// </summary>
+        public void DoStuff(int a)
+        {
+
+        }
+
+        /// <summary>
+        /// This method does more stuff
+        /// </summary>
+        public void DoMoreStuff(int a, string b)
+        {
+
+        }
+    }
+
+<a name="seealso"></a>
+## &lt;seealso&gt;
+
+The &lt;seealso&gt; tag lets you specify the text that you might want to appear in a See Also section.
+
+    /// <summary>
+    /// This class does something
+    /// </summary>
+    public class SomeClass
+    {
+        /// <summary>
+        /// This method does stuff
+        /// <seealso cref="SomeClass.DoMoreStuff" />
+        /// </summary>
+        public void DoStuff(int a)
+        {
+
+        }
+
+        /// <summary>
+        /// This method does more stuff
+        /// </summary>
+        public void DoMoreStuff(int a, string b)
+        {
+
+        }
+    }
+
+<a name="param"></a>
+## &lt;param&gt;
+
+The &lt;param&gt; tag should be used in the comment for a method declaration to describe one of the parameters for the method.
+To document multiple parameters, use multiple &lt;param&gt; tags.
+
+    /// <summary>
+    /// This class does something
+    /// </summary>
+    public class SomeClass
+    {
+        /// <param name="a">The stuff to do</param>
+        public void DoStuff(int a)
+        {
+
+        }
+
+        /// <param name="a">A stuff to do</param>
+        /// <param name="b">Another stuff to do</param>
+        public void DoMoreStuff(int a, string b)
+        {
+
+        }
+    }
+
+The `name` parameter represents the name of a method parameter
+
+<a name="paramref"></a>
+## &lt;paramref&gt;
+
+The &lt;paramref&gt; tag gives you a way to indicate that a word in the code comments, 
+for example in a &lt;summary&gt; or &lt;remarks&gt; block refers to a parameter.
+
+    /// <summary>
+    /// This class does something
+    /// </summary>
+    public class SomeClass
+    {
+        /// <summary>
+        /// This method does stuff.
+        /// The <paramref name="a"/> takes a number
```

Comment:
This would get rendered as something like "The *a* takes a number". That doesn't sound like a proper sentence to me. I think a better way would be:

```c#
/// The parameter <paramref name="a"/> takes a number 
```